### PR TITLE
Set up config file for local cluster development

### PR DESCRIPTION
**Pull request by Shillaker**
Adds `--local` option to `knative.ini-file` task for running against a local cluster.

### CODE REVIEW

File: bin/cluster_dev.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ docker-compose -f docker-compose.yml \
     up \
     --no-recreate \
     -d \
+    --scale worker=2
```
Comment (Shillaker) [inline bin/cluster_dev.sh:35]: Wasn't actually a cluster before as it was only running one worker.

File: docker-compose.yml [hunk 2]
```diff
@@ -64,6 +64,7 @@ services:
       - NETNS_MODE=off
       - REDIS_QUEUE_HOST=redis-queue
       - REDIS_STATE_HOST=redis-state
+      - OVERRIDE_CPU_COUNT=4
```
Comment (Shillaker) [inline docker-compose.yml:67]: Setting a lower CPU count on each worker forces function executions to be distributed. Can be adjusted for smaller hosts too.

File: logs/worker-1-log [hunk 3]
```diff
@@ -0,0 +1,431 @@
+[33mworker_1       |[0m Setting up cgroup
```
Comment (csegarragonz) [inline logs/worker-1-log:1]: I think these two log files accidentaly made it to the PR.

**Review by csegarragonz — CHANGES_REQUESTED** [review]
LGTM, just make sure to delete the `logs/*` files

File: logs/worker-1-log (hunk 3 above)
Comment (Shillaker) [reply logs/worker-1-log:1]: Ah yes, whoops!

**Review by csegarragonz — APPROVED** [review]

**Pull request merged**